public/index.html

```diff
@@ -27,10 +27,8 @@
   </head>
 
   <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
```

Comment:
Why is this changed? We're automatically formatting this project using Prettier. Please run `npm run format`, and it will be consistent across the codebase.

Reply:
I did run `npm run format`, and it changed the above few lines to be on one line, so I reverted it back manually to stay consistent.